database/seeders/DatabaseSeeder.php

```diff
@@ -14,11 +14,7 @@ class DatabaseSeeder extends Seeder
      */
     public function run()
     {
-        // \App\Models\User::factory(10)->create();
-
-        // \App\Models\User::factory()->create([
-        //     'name' => 'Test User',
-        //     'email' => 'test@example.com',
-        // ]);
+        $this->call(RoleSeeder::class);
+        $this->call(AdminSeeder::class);
```

Comment:
If we gonna add them to the Database seeder that means each time we run `db:seed`  we will add a new role, either

1. We Remove it from Database Seeder and we use individual seeders when we need them.
2. We add login to the seeder not to add the columns again if case it exsits